js/ui/layout.js

```diff
@@ -63,7 +63,8 @@ const defaultParams = {
 };
 
 var LayoutManager = GObject.registerClass({
-    Signals: { 'hot-corners-changed': {},
+    Signals: { 'background-clicked': {},
```

Comment:
This is OK, but I'm not sure that's all we need.
As mentioned on the phabricator ticket, in the eos3.5 codebase `sideComponent.js` connects to this signal.
This patch will fix the warning when the signal is emitted, but functionality-wise, on master, even after this patch I don't see anything connecting to this signal.

Reply:
You're right; brought back the code that connected to it.